prover/src/composer/mod.rs

```diff
@@ -88,7 +88,7 @@ impl<E: FieldElement> DeepCompositionPoly<E> {
                 &mut t1_composition,
                 poly,
                 ood_trace_states[0][i],
-                self.cc.trace[i].0,
```

Comment:
I think we should also update comments on lines 57 - 58 above.

Reply:
Thanks